docs/installation/configuration/cmake_options.rst

```diff
@@ -0,0 +1,173 @@
+          on Fast DDS TLS configuration.
+        - ``OFF``
+    *   - :class:`COMPILE_EXAMPLES`
+        - Builds the Fast DDS examples. It is set to ``ON`` if :class:`EPROSIMA_BUILD` is ``ON`` and |br|
+          :class:`EPROSIMA_INSTALLER` is ``OFF``. These examples can be found in the
+          `eProsima Fast DDS <https://github.com/eProsima/Fast-DDS/tree/master/examples>`_ |br|
+          `GitHub repository <https://github.com/eProsima/Fast-DDS/tree/master/examples>`_.
+        - ``OFF``
+    *   - :class:`BUILD_DOCUMENTATION`
+        - Uses doxygen to create the Fast DDS documentation. It is set to ``ON`` if :class:`EPROSIMA_INSTALLER` |br|
+          is ``ON`` or if :class:`CHECK_DOCUMENTATION` is ``ON``.
+        - ``OFF``
+    *   - :class:`CHECK_DOCUMENTATION`
+        - Uses doxygen to check Fast DDS code documentation.
+        - ``OFF``
+    *   - :class:`INSTALL_EXAMPLES`
+        - Installs the Fast DDS examples. It is set to ``ON`` if :class:`EPROSIMA_INSTALLER` is ``ON``.
+        - ``OFF``
+    *   - :class:`STRICT_REALTIME`
+        - Enables a strict real-time behaviour. Please refer to the Real-Time Use Case for |br|
+          more information on Fast DDS real-time configuration.
+        - ``OFF``
+    *   - :class:`INTERNAL_DEBUG`
+        - Activate developer debug messages. It is set to ``ON`` if :class:`EPROSIMA_BUILD` is ``ON``.
+        - ``OFF``
+
+Third-party libraries options
+-----------------------------
+
+Fast DDS relies on the `eProsima FastCDR <https://github.com/eProsima/Fast-CDR>`_ library for serialization
+mechanisms.
+Moreover, Fast DDS requires two internal third-party libraries for its proper operation: Asio and TinyXML2.
+Asio is a cross-platform C++ library for network and low-level I/O programming; while TinyXML2 parses the XML profile
+files, for later translation into C++ source code.
+These libraries can also be configured using Fast DDS CMake options.
+
+.. list-table::
+    :header-rows: 1
+
+    *   - Option
+        - Description
+        - Default
+    *   - :class:`THIRDPARTY`
+        - Activates the use of Fast CDR and the internal third-party libraries. It is set to ``ON`` |br|
+          if EPROSIMA_BUILD is ``ON``.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_UPDATE`
+        - Activates the automatic update of Fast CDR and the internal third-party libraries.
+        - ``ON``
+    *   - :class:`THIRDPARTY_fastcdr`
+        - Activates the use of Fast CDR library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_Asio`
+        - Activates the use of Asio internal third-party library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_TinyXML2`
+        - Activates the use of TinyXML2 internal third-party library.
+        - ``OFF``
+    *   - :class:`THIRDPARTY_android-ifaddrs`
+        - Activates the use of android-ifaddrs, an implementation of :func:`getifaddrs` for Android. |br|
+          Only if :class:`ANDROID` is set to ``ON``.
+        - ``OFF``
+
+Test options
```

Comment:
As a general comment for the subsection, I think we could combine all the options into a single table. What do you think?